CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to Quiz
+
+We love your input! We want to make contributing to this project as easy and transparent as possible.
+
+## Ways to Contribute
+
+- Bug Reports
+- Submitting fixes
+- Discussing the current state of the code
+- Feature Requests: Proposing new features
+- Become a maintainer
+
+## How to Contribute
+
+### Making Changes
+
+1. Fork the repository.
+2. Create a new branch (git checkout -b feature-branch).
+3. Make your changes and ensure code quality (linting, tests).
+3. Commit your changes (git commit -am 'Add new feature').
+3. Push to the branch (git push origin feature-branch).
+3. Create a new Pull Request.
+
+### How to Start the Project
+
+1. After forking, clone the repo onto your local machine (git clone repo-name).
+2. `index.html` is the starting point of the website.
+3. The music, css, js and images are in their respective folders.
+
+### Pull Requests (PRs)
+
+- Clearly describe your changes in the PR description.
+- Ensure the PR addresses an open issue (if applicable).
+- Maintain a clean commit history.
+  
+### Code Style
+
+- Follow the existing code style and conventions.
+- Document your code and changes wherever necessary.
+
+### Issue Reporting
+
+- Report a bug by opening a new [issue](https://github.com/KendallDoesCoding/quiz/issues/new/choose).
+- When reporting bugs, provide detailed steps to reproduce.
+- Include relevant information such as OS, browser, or environment.
+
+## Community Guidelines
+
+Be respectful and considerate towards others' contributions.
+Accept constructive criticism and feedback gracefully.
+Help fellow contributors and maintain a positive environment.
+
+## Licensing
+
+By contributing, you agree that your contributions will be licensed under the project's [LICENSE](https://github.com/KendallDoesCoding/quiz/blob/main/LICENSE).
+
+## Contact
+
+If you have any questions or need further assistance, feel free to contact us at <kendall@kendalldoescoding.tech>.
```

Comment:
```suggestion
If you have any questions or need further assistance, feel free to contact us at <kendalldoescoding@gmail.com>.
```

for now, the domain expired, will get it back soon, so put this email.

Reply:
ok i'll make the required changes.